File: Templates/Go/classes/ClassWithInheritance.go
type ParentClass struct {
    // Define properties and methods here
}

type ChildClass struct {
    ParentClass
    additionalProperty Type
}

func NewChildClass(property1 Type, property2 Type, additionalProperty Type) *ChildClass {
    return &ChildClass{ParentClass{property1, property2}, additionalProperty}
}

File: Templates/Java/classes/ClassWithInheritance.java
public class ParentClass {
    // Define properties and methods here
}

public class ChildClass extends ParentClass {
    private Type additionalProperty;

    public ChildClass(Type property1, Type property2, Type additionalProperty) {
        super(property1, property2);
        this.additionalProperty = additionalProperty;
    }
}
